www-client/lynx/lynx-2.9.0_pre12-r2.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/thomasdickey.asc
+inherit verify-sig flag-o-matic
+
+case ${PV} in
+	*_pre*) MY_P="${PN}${PV/_pre/dev.}" ;;
+	*_rc*)  MY_P="${PN}${PV/_rc/pre.}" ;;
+	*_p*|*) MY_P="${PN}${PV/_p/rel.}" ;;
+esac
+
+DESCRIPTION="An excellent console-based web browser with ssl support"
+HOMEPAGE="https://lynx.invisible-island.net/"
+SRC_URI="https://invisible-mirror.net/archives/${PN}/tarballs/${MY_P}.tar.bz2"
+SRC_URI+=" verify-sig? ( https://invisible-mirror.net/archives/${PN}/tarballs/${MY_P}.tar.bz2.asc )"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
+IUSE="brotli bzip2 cjk gnutls idn nls ssl"
+
+RDEPEND="
+	sys-libs/ncurses:=[unicode(+)]
+	sys-libs/zlib
+	brotli? ( app-arch/brotli:= )
+	bzip2? ( app-arch/bzip2 )
+	idn? ( net-dns/libidn2:= )
+	nls? ( virtual/libintl )
+	ssl? (
+		!gnutls? (
+			dev-libs/openssl:=
+		)
+		gnutls? (
+			dev-libs/libgcrypt:=
+			>=net-libs/gnutls-2.6.4:=
+		)
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+	verify-sig? ( sec-keys/openpgp-keys-thomasdickey )
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.9.0_pre9-mint.patch"
+	"${FILESDIR}/${PN}-2.9.0_pre9-parallel.patch"
+)
+
+src_configure() {
+	append-cppflags -D_GNU_SOURCE
```

Comment:
1. This needs a bug reference above it in a comment.
2. This needs some rationale/justification. Why aren't we fixing the build system properly to do this for us? (A valid answer is: you think this might be fixed in git, but it's not yet in a release. I think that might be the case here from looking at it...)